<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KgvPageLayoutHeaderBase</class>
  <widget class="QWidget" name="KgvPageLayoutHeaderBase" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>445</width>
        <height>437</height>
      </rect>
    </property>
    <layout class="QVBoxLayout" >
      <item>
        <widget class="QGroupBox" name="groupBox1" >
          <property name="title" >
            <string>Header</string>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <widget class="QCheckBox" name="rhFirst" >
                <property name="text" >
                  <string>Different header for the first page</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="rhEvenOdd" >
                <property name="text" >
                  <string>Different header for even and odd pages</string>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <spacer name="spacer2" >
                    <property name="sizeHint" >
                      <size>
                        <width>21</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <widget class="QLabel" name="textLabel3" >
                    <property name="text" >
                      <string>Spacing between header and body:</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QFrame" name="headerSpacingPane" >
                    <property name="frameShape" >
                      <enum>NoFrame</enum>
                    </property>
                    <property name="frameShadow" >
                      <enum>Raised</enum>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="groupBox2" >
          <property name="title" >
            <string>Footer</string>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <widget class="QCheckBox" name="rfFirst" >
                <property name="text" >
                  <string>Different footer for the first page</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="rfEvenOdd" >
                <property name="text" >
                  <string>Different footer for even and odd pages</string>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <spacer name="spacer3" >
                    <property name="sizeHint" >
                      <size>
                        <width>41</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item>
                  <widget class="QLabel" name="textLabel4" >
                    <property name="text" >
                      <string>Spacing between footer and body:</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QFrame" name="footerSpacingPane" >
                    <property name="frameShape" >
                      <enum>NoFrame</enum>
                    </property>
                    <property name="frameShadow" >
                      <enum>Raised</enum>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="groupBox3" >
          <property name="title" >
            <string>Footnote/Endnote</string>
          </property>
          <layout class="QHBoxLayout" >
            <item>
              <spacer name="spacer4" >
                <property name="sizeHint" >
                  <size>
                    <width>40</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Horizontal</enum>
                </property>
              </spacer>
            </item>
            <item>
              <widget class="QLabel" name="textLabel5" >
                <property name="text" >
                  <string>Spacing between footnote and body:</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QFrame" name="footnotePane" >
                <property name="frameShape" >
                  <enum>NoFrame</enum>
                </property>
                <property name="frameShadow" >
                  <enum>Raised</enum>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
